shared/src/models/patient.ts

```diff
@@ -139,21 +119,22 @@ export class Patient {
      */
     @IsNumber()
     @Min(0)
-    public readonly timeSpeed: number = 1;
+    public readonly changeSpeed: number = 1;
 
     /**
-     * This can be any arbitrary string. It gives trainers the freedom to add additional functionalities that are not natively supported by this application (like an hospital ticket system)
+     * The time a patient has been treated for
      */
-    @IsString()
-    @MaxLength(65535)
-    public readonly remarks: string;
-
```

Comment:
This should not get deleted

Reply:
I think this only got moved and not removed.